src/services/AnalysisArgumentParser.ts

```diff
@@ -15,25 +16,28 @@ const getIntegrateUrl = (scanType: ScanType) =>
     scanType == ScanType.CSA ? "containers" : scanType.toLowerCase()
   }`;
 
-interface IBaseScanArguments {
-  apiKey: string;
-  apiURL: string;
+interface IBaseScanArguments extends ICommonArguments {
   appVersion: string;
   branchName: string;
   branchURI: string;
   buildURI: string;
   buildVersion: string;
-  clientId: string;
   commitHash: string;
   contributingDeveloperId: string;
   contributingDeveloperSource: ContributingDeveloperSource;
   contributingDeveloperSourceName: string;
   integrationName: IntegrationName;
   integrationType: IntegrationType;
-  logLevel: LogLevel;
   onFailure: OnFailure;
   operatingEnvironment: string;
   projectName: string;
+}
+
+interface ICommonArguments {
```

Comment:
organizationName and secret are common as well (which would be the github org and github pat for the github implementation)

Reply:
this is for the arg parse, they are not common with the scans

Reply:
they are common for the scm audit params - devops/bitbucket etc would also have an organizationName and secret for requests

Reply:
chatted on slack - the intent is for these to be common across all services